frontend/src/component/Search/AdvancedSearchContentList.jsx

```diff
@@ -0,0 +1,136 @@
+        ...props.contentType.find(ct => ct.slug === searchItem.contentType) || { hexcolor: '', label: '', faIcon: '' },
+        status: props.contentType.find(ct => ct.slug === searchItem.contentType).availableStatuses.find(s => s.slug === searchItem.status) || { hexcolor: '', label: '', faIcon: '' }
+      }
+    }
+  })
+
+  return (
+    <>
+      <div className='content__header'>
+        <div className='advancedSearchContent__type__header'>
+          {props.t('Type')}
+        </div>
+        <div className='advancedSearchContent__title'>
+          {props.t('Title and path')}
+        </div>
+        <div className='advancedSearchContent__typeAndTitle'>
+          {props.t('Title')}
+        </div>
+        <div className='advancedSearchContent__modification'>
+          {props.t('Last modification')}
+        </div>
+        <div className='advancedSearchContent__information'>
+          {props.t('Information_plural')}
+        </div>
+      </div>
+
+      {resultList.map((searchItem, index) => (
+        <ListItemWrapper
+          label={searchItem.label}
+          read
+          contentType={searchItem.contentType}
+          isLast={index === resultList.length - 1}
+          key={searchItem.contentId}
+        >
+          <Link
+            to={`${PAGE.WORKSPACE.CONTENT(searchItem.workspaceId, searchItem.contentType.slug, searchItem.contentId)}`}
+            className='advancedSearchContent'
+          >
+            <div
+              className='advancedSearchContent__type__content'
+              style={{ color: searchItem.contentType.hexcolor }}
+              title={props.t(searchItem.contentType.label)}
+            >
+              <Icon
+                icon={`fa-fw ${searchItem.contentType.faIcon}`}
+                color={searchItem.contentType.hexcolor}
+              />
+              <span>{props.t(searchItem.contentType.label)}</span>
+            </div>
+
+            <div
+              className='advancedSearchContent__title'
+              title={searchItem.label}
+            >
+              {searchItem.label}
+              {searchItem.contentType.slug === CONTENT_TYPE.FILE && (
+                <Badge text={searchItem.fileExtension} customClass='badgeBackgroundColor' />
+              )}
+            </div>
+
+            {searchItem.contentType.slug !== CONTENT_TYPE.FOLDER && (
+              <div className='advancedSearchContent__information'>
+                <span title={props.t(searchItem.contentType.status.label)}>
+                  {props.t(searchItem.contentType.status.label)}
+                </span>
+                <Icon
+                  icon={`fa-fw ${searchItem.contentType.status.faIcon}`}
+                  title={props.t(searchItem.contentType.status.label)}
+                  color={searchItem.contentType.status.hexcolor}
+                />
+                <span
+                  title={props.t('{{numberComments}} comments', { numberComments: searchItem.commentCount })}
```

Comment:
I think this makes it too complex, since I will create a const inside a map that is inside the render (and a little bit far from where it is used). I understand not duplicating the code, but in this case if it doesn't bother you too much I'd rather leave it like this.